cosmos/operators/virtualenv.py

```diff
@@ -140,7 +144,7 @@ def execute(self, context: Context, **kwargs: Any) -> None:
     def on_kill(self) -> None:
         self.clean_dir_if_temporary()
 
-    def _prepare_virtualenv(self) -> str:
+    def _prepare_virtualenv(self) -> Any:
```

Comment:
mypy complained about this here: https://github.com/astronomer/astronomer-cosmos/actions/runs/14517759820/job/40730877187